x-pack/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js

```diff
@@ -72,9 +72,13 @@ export class TooltipControl extends React.Component {
         }
       }
       if (!match) {
+        //This keeps track of first properties (assuming these will be identical for features in different tiles
         uniqueFeatures.push({
           id: featureId,
           layerId: layerId,
+          meta: {
```

Comment:
Why are you wrapping `mbProperties` in `meta`? Do you see other uses for `meta`?

I think it would be clearer to just make `mbProperties` a top level member of `TooltipFeature`. Then, they tooltips can use be used `mbProperties` as a fallback if `layer.getFeatureById` returns null. That way, more of the logic for using `mbProperties` stays local to tooltip components that use them.

Reply:
Discussed.

Moving mbProperties out of the `getFeatureById` method. `TooltipPopover` will now provide the fallback. When the `ILayer#getFeatureById` returns null, `TooltipPopover` will use `mbProperties` as a fallback.